samples/v1.5/Scenarios/WeatherLarge.json

```diff
@@ -97,7 +97,7 @@
 					],
 					"selectAction": {
 						"type": "Action.OpenUrl",
-						"title": "View Wednesday",
+						"title": "View Wednesday, Drizzly weather. Highest temperature 50 degrees. Lowest temperature 32 degrees.",
```

Comment:
Can you update tooltip with the new addition? Changing title will change the look of card.

Reply:
I can update it to tooltip if you want for clarity, but `title` defaults to `tooltip` on select action. Both have the same effect

<img width="479" alt="image" src="https://github.com/microsoft/AdaptiveCards/assets/98650930/32768609-423b-41d4-b604-4d7756ad8471">

Reply:
@anna-dingler, thanks for the image, that brought the clarification. the change looks good.  Approved.